### PR TITLE
fix variable overlap

### PR DESCRIPTION
**Pull request by liyantang**
Fix overlapping variable names and line length too long.

### CODE REVIEW

File: src/blockInlineStyles.js [hunk 1]
```diff
@@ -64,28 +64,29 @@ const isPartiallyIntersectingMutation = (entities, style) => {
   });
 };
 
-const shouldResetStyles = (entities, remainingStyles, currentIndex, resetStyle) => {
+const needResetStyles = (entities, remainingStyles, currentIndex, resetStyle, i) => {
```
Comment (kedromelon) [inline src/blockInlineStyles.js]: is there a better name to give `i` in the scope of this function? i'm having a hard time being clear on what exactly it represents here

Comment (liyantang) [inline src/blockInlineStyles.js]: Umm, I guess we can name it remainingStylesIndex even thought it's wordy.

Comment (kedromelon) [inline src/blockInlineStyles.js]: let's go with that for now -- i think it's much more clear.

more optional, but i wonder if we should also rename `currentIndex` to more clearly differentiate it from `remainingStylesIndex` too?

Comment (liyantang) [inline src/blockInlineStyles.js]: Hmm....well currentIndex represent the index of rawBlock text as it's iterating through each char.  Any suggestions?

Comment (kedromelon) [inline src/blockInlineStyles.js]: i'm not sure, that's a tough one 😛 
let's just leave that as-is for now

**Pull request merged**